invenio/modules/news/templates/admin/Edit.html

```diff
@@ -0,0 +1,20 @@
+{#
```

Comment:
- [ ] all files should be lowercased
- [ ] rename to `invenio/modules/news/templates/news/edit.html`